style.css

```diff
@@ -0,0 +1,53 @@
+body{
+    text-align: center;
+    background-color: grey;
+}
+
+body > h1 >p{
+    color: white;
+    
+}
+
+.form-page{
+    
+    width: 600px;
+    padding: 20px;
+    margin: 50px 30%;
+    background-color: white;
+    text-align: left;
+}
+
+
+
```

Comment:
> Add a professional Readme.md for your project and well documented commit message

- [ ] Kindly make this comment polite too for reasons I earlier pointed

Reply:
- [ ] [OPTIONAL] So far you have done incredible work 👍 👏 . You made impresive changes required by the previous code reviewer. However, i suggest that you pay more attention to giving detailed explanation on why the changes you required are important 😥 . This will help the student have a clear understanding of what you are referring to.
- [ ] [OPTIONAL] In addition, you should try as much as possible to include the imojis in your comments. This makes the student fell more comfortable because you comments sound more friendly ☺️ .